### PR TITLE
jsdoc: add Orders, Shipping, Payment methods from core

### PR DESCRIPTION
**Pull request by machikoyasuda**
## What this PR does
1. Adds `Methods/Orders`
2. Adds `Methods/Shipping`
3. Adds `Methods/Payment`

## What that renders
screen shot 2017-10-24 at 6 30 19 pm

### CODE REVIEW

File: server/methods/core/orders.js [hunk 1]
```diff
@@ -9,18 +9,36 @@ import { SSR } from "meteor/meteorhacks:ssr";
...
+ *
+ * @namespace Methods/Orders
+*/
 
-// helper to return the order credit object
-// credit paymentMethod on the order as per current active shop
-// returns entire payment method
+/**
+ * @name orderCreditMethod
+ * @summary Helper to return the order credit object.
```
Comment (spencern) [inline server/methods/core/orders.js]: Does this one need a `memberof Methods/Orders` tag?

File: server/methods/core/shipping.js [hunk 2]
```diff
@@ -4,6 +4,22 @@ import { Cart, Accounts } from "/lib/collections";
...
+ * @file Methods for Shipping - methods typically used for checkout (shipping, taxes, etc).
+ * Run these methods using `Meteor.call()`.
+ *
+ *
+ * @namespace Methods/Shipping
+*/
+
+/**
+ * @name createShipmentQuotes
+ * @private
```
Comment (spencern) [inline server/methods/core/shipping.js]: Do we need a summary for this private method?

File: server/methods/core/shipping.js [hunk 3]
```diff
@@ -54,9 +70,11 @@ function createShipmentQuotes(cartId, shopId, rates) {
   return update;
 }
 
-
 /**
- * @summary if we have items in the cart, ensure that we only have shipping records for shops currently represented in the cart
+ * @name pruneShippingRecordsByShop
```
Comment (spencern) [inline server/methods/core/shipping.js]: Guessing we're not using the `@membeof` here because it's private?

Comment (machikoyasuda) [reply server/methods/core/shipping.js]: Correct. `@private` methods don't need `@memberof`. But they should still have `@summary` for people to read in the code itself.

File: server/methods/core/orders.js (hunk 1 above)
Comment (machikoyasuda) [reply server/methods/core/orders.js]: ✔️

File: server/methods/core/shipping.js (hunk 2 above)
Comment (machikoyasuda) [reply server/methods/core/shipping.js]: Not required but good to have.

**Comment by machikoyasuda** [comment]
Updated for comments + updated master branch

**Comment by spencern** [comment]
eslint passes locally.

**Pull request merged**